models/src/main/protobuf/CasperMessage.proto

```diff
@@ -75,6 +75,7 @@ message DeployData {
   int64 phloPrice     = 7; //phlo price
   int64 phloLimit     = 8; //phlo limit for the deployment
   int32 nonce         = 9; //nonce for transaction made against `from` wallet
+  int64 validAfterBlockNumber = 10;
```

Comment:
The deploy is already valid for that block number, right? If so, the naming might be a little confusing.

Reply:
Hmm but `validAfterOrOnBlockNumber` is a bit long. I think I'll just change the code so it only validates if it is strictly after.